### PR TITLE
Information Security 101

### PR DESCRIPTION
**Pull request by BaiaVieira**
Information Security 101
=========================

* Speaker   : Basilio Vieira
* Available : Full availability
* Length    : 45-60 minutes
* Language  : English

Description
-----------

Learn how to fuel your organisation safety culture from Day 1. Let’s take a holistic approach to the challenges on Information Security looking into industry best practices, management systems, Governance, risks and certifications. In the end, there will be cake.

This is a talk on Confidentiality, Integrity and Availability of Information and the benefits and losses arising from the way an organisation deals with it.

The topics:

Industries Best Practices

* Passwords - If it ain't broke, don't fix it!
* 2 factor authentication - The good, the bad, the ugly
* Why the clear screen / clear desk policy

Management systems

* A cookie factory example
* What are they and ISO 9001
* Deming Cycle (Plan, Do, Check, Act)
* Risk Assessment / Risk Treatment

ISO 27001 Information Security Management System

* Confidentiality, Integrity, Availability
* Secure software development
* Business Continuity - Resilience

GDPR

* Past history on Data Protection
* Data breaches before and after GDPR
* Impact of rights for data subjects
* Common myths
* The Google 50 million fine - a slap in the wrists
* iOS / Android - different philosophies
* The future

This will gravitate around the themes of Portal, Koolaid, Lost, Super Mario Bros and Brexit.

#TheCakeIsALie

Speaker Bio
-----------
Developer, Maker, Geek now Auditor.

Links
-----

* Blog: www.basilio.eu
* Company: www.bureauveritas.com
* GitHub: https://github.com/BaiaVieira

Extra Information
-----------------
Although largely centred on my current work, this is not a sales pitch.

### CODE REVIEW

**Comment by edevil** [comment]
@BaiaVieira Thank you for the talk proposal. Could you elaborate a bit more on the talk subject, maybe add a list of topics that you want to address?

**Comment by BaiaVieira** [comment]
Thank you for considering this talk.

This is a talk on Confidentiality, Integrity and Availability of Information and the benefits and losses arinsing from the way an organisation deals with it.

I will elaborate on the following topics:

Definitions

Risk Assessment / Risk Treatment

Management systems
- What are they and ISO 9001
- Deming Cycle (Plan, Do, Check, Act)
- A cookie factory example 

ISO 27001 Information Security Managment System
- Confidentiality, Integrity, Availability
- Secure software development
- Resilience

Industries Best Practices
- Passwords
- 2 factor authentication
- Why the clear screen / clear desk policy

GDPR
- Past history on Data Protection
- Impact of rights for data subjects
- Common myths
- The Google 50 million fine
- The future
    
This will gravitate around the themes of Portal, Koolaid, Lost, Super Mario Bros and Brexit.

... and it's still a work in progress.

**Comment by carlosefr** [comment]
Ok, can you extend the proposal itself (and the PR description along with it) with some of that additional information?

File: Information-Security-101_Basilio-Vieira.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+* Language  : English
+
+Description
+-----------
+
+Learn how to fuel your organisation safety culture from Day 1. Let’s take a holistic approach to the challenges on Information Security looking into industry best practices, management systems, Governance, risks and certifications. In the end, there will be cake.
+
+This is a talk on Confidentiality, Integrity and Availability of Information and the benefits and losses arising from the way an organisation deals with it.
+
+The topics:
```
Comment (BaiaVieira) [inline Information-Security-101_Basilio-Vieira.md:16]: Added topics.

**Comment by BaiaVieira** [comment]
Done! I guess... Maybe... Yup, I believe I pressed all the correct buttons.
#DontReallyKnow

**Pull request merged**